examples/research_projects/unilm/layoutlmv2/requirements.txt

```diff
@@ -0,0 +1,3 @@
+transformers>=4.12.2
+datasets>=1.8.0
+git+https://github.com/facebookresearch/detectron2.git
```

Comment:
This also needs `accelerate` for the `no_trainer` version.

Reply:
Understood.